packages/runtime/src/components/shared/BackgroundsContainer/index.tsx

```diff
@@ -25,15 +25,18 @@ export default forwardRef<HTMLDivElement | null, Props>(function BackgroundsCont
   { backgrounds, children, className, ...restOfProps }: Props,
   ref: Ref<HTMLDivElement>,
 ) {
+  const widthAndMarginStyle = useStyle({
+    width: '100%',
+    margin: '0 auto',
+  })
+
   return (
     <div
       {...restOfProps}
       ref={ref}
       className={cx(
```

Comment:
Note that Emotion's [`cx` docs](https://emotion.sh/docs/@emotion/css#cx) say that `Emotion generated styles are applied from left to right. Subsequent styles overwrite property values of previous styles.`, but this is clearly not the case here.

We could dig deeper in why not, but personally I'd argue that hard-coding defaults for CSS properties that will have their true defaults generated from prop values is redundant and makes it harder to understand how we arrive at the final component styles, and therefore needs to go either way.

Reply:
From offline: Btw, I did a lot more work that I didn't surface in my PR. I actually tested various fixes to see how the injected class order changes (I kept these test results saved, if you're interested). It turns out that while `cx` may follow the left to right rule, what's making the impact is the fact that `useStyles` has a side effect that injects classes into the cache directly, which directly determines the order they're flushed. This flush order is what we're concerned with - so if you've already generated a class and pass it to `cx`, it's too late, it's already in the cache.